src/late_multicellular_stage/editor/MetaballEditorComponentBase.cs

```diff
@@ -721,8 +717,36 @@ protected void RunWithSymmetry(Vector3 position, TMetaball? parent, Action<Vecto
                 break;
             }
 
+            case HexEditorSymmetry.XAxisSymmetry:
+            {
+                var symmetryPosition = position * new Vector3(-1, 1, 1);
+
+                if (position.DistanceTo(symmetryPosition) < diameter / 2)
+                {
+                    callback(position, parent);
+                }
+                else
+                {
+                    callback(position, parent);
+
+                    if (parent != null)
+                    {
+                        var symmetryParent = (TMetaball?)editedMetaballs.GetClosestMetaballToPosition(
```

Comment:
This seems like a place that could be done more type safely.

I think I'll have time tomorrow to fix up the few small things left in this if they aren't done by then yet.

Reply:
I'll commit a fix for this, was just a case of using `T` as the return type in the generic type to return the real metaball type to remove the need for this cast.